marshal.go

```diff
@@ -27,7 +27,7 @@ type pbLink struct {
 func Marshal(inNode ipld.Node, out io.Writer) error {
 	// Wrap in a typed node for some basic schema form checking
 	builder := Type.PBNode.NewBuilder()
-	if err := builder.AssignNode(inNode); err != nil {
+	if err := builder.ConvertFrom(inNode); err != nil {
```

Comment:
fyi, we'll probably undo this rename, but it should be easy for you to undo it here too.